google-cloud-storage/acceptance/storage/file_test.rb

```diff
@@ -316,6 +319,34 @@
     uploaded.delete
   end
 
+  it "should download, verify, and decompress a gzipped file with default md5 verification" do
+    skip "Not working. See https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/1835"
+
+    lazy_bucket = storage.bucket bucket_public_test_name
+    lazy_file = lazy_bucket.file file_public_test_gzip_name
+
+    Tempfile.open ["google-cloud", ".png"] do |tmpfile|
```

Comment:
What's the .png for here? (Sorry, I don't know Ruby at all...)

Reply:
Oops, that's a copy/paste from another test... it doesn't matter, but is not appropriate. Thanks.